addons/im_livechat/static/src/core/channel_member_list_patch.xml

```diff
@@ -5,9 +5,9 @@
             <div class="d-flex flex-column">
                 <t>$0</t>
                 <div class="ms-2 d-flex">
-                    <span t-if="member.persona?.lang_name" class="me-2">
+                    <span t-if="channelMemberService.getLang(member)" class="me-2">
                         <i class="fa fa-comment-o me-1" aria-label="Lang"/>
-                        <t t-esc="member.persona.lang_name"/>
+                        <t t-esc="channelMemberService.getLang(member)"/>
```

Comment:
I would be better as a getter `langName` on `MemberModel`.
Would simply be `member.langName` here.

_I get there's value in keeping model small, but I don't think it's worth the cost of readability in template (and coupling model to a specific service, because service !== model although we badly named and split them ATM). Sometimes it's motivated to do so when it requires some other services(e.g. `MessageService@dateSimple`, which requires `UserService`), but that's not the case here._

Reply:
Moved to the model. Note that it is still a function since it has to be patched.